pkg/upstream/admin-console.go

```diff
@@ -32,6 +32,29 @@ type UpstreamSettings struct {
 	IncludeMinio           bool
 }
 
+func WriteAdminConsoleFiles(renderDir string, options types.WriteOptions) error {
```

Comment:
renderDir won't make sense since we're not really rendering anything, maybe:
```suggestion
func WriteAdminConsoleFiles(outputDir string, options types.WriteOptions) error {
```

Reply:
to be fair, it kinda is rendering YAML and `renderDir` is used throughout the admin console generation code. so i'm probably keeping it to be consistent with that.